SPECS/cloud-init/cloud-init.spec

```diff
@@ -176,7 +176,7 @@ make check %{?_smp_mflags}
 - Remove add-mariner-distro-support and CVE-2022-2084 pathc as no longer needed in newer version
 
 * Tue Oct 04 2022 Minghe Ren <mingheren@microsoft.com> - 22.2-9
-- add BuildRequires mariner-release to make sure /etc/os-release exists so variant can be set as mariner properly
+- add BuildRequires azurelinux-release to make sure /etc/os-release exists so variant can be set as mariner properly
```

Comment:
That changelog entry was correct. At that time it was called `mariner-release`. No need to change it.

Reply:
Updated